jwst/skymatch/region.py

```diff
@@ -36,18 +36,17 @@ def __str__(self):
         return self._message
 
 
-class Region():
+class Region(object):
     """
     Base class for regions.
 
     Parameters
     -------------
-    rid : int or string
+    rid: int or string
```

Comment:
Not sure why this change but the entire docstring needs reformatting.
`---` should be the length of `Parameters` string
a space between `rid` and `:` is necessary (here and in all other docstrings)
the type should be `int or str`